dashboards/grafana-dashboard-network-verifier.configmap.yaml

```diff
@@ -0,0 +1,653 @@
+            "mode": "markdown"
+          },
+          "pluginVersion": "9.0.3",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "tuHy3WB7z"
+              },
+              "refId": "A"
+            }
+          ],
+          "title": "Playground usage",
+          "type": "text"
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": {
+            "type": "prometheus",
+            "uid": "gH4yqWBnz"
+          },
+          "description": "The Network verifier Failure percentage over the last 7 days",
+          "fieldConfig": {
+            "defaults": {
+              "links": []
+            },
+            "overrides": []
+          },
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 7,
+            "w": 5,
+            "x": 0,
+            "y": 10
+          },
+          "hiddenSeries": false,
+          "id": 2,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "links": [],
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "9.0.3",
+          "pointradius": 5,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "gH4yqWBnz"
+              },
+              "expr": "(sum(network_verifier_failures{}) / sum(network_verifier_runs)) * 100",
```

Comment:
just for knowledge: how are those metrics being populated?
are those from ocm backend? if so why do we integrate it into verifier's repo instead of the consumer service repo?

Reply:
those we built with the grafana playground. following the documentation for app-interface here : https://gitlab.cee.redhat.com/service/app-interface#add-a-grafana-dashboard 

Also the reason why we add it here is on the recommendation of the app-sre team the conversation is on the slack thread I put on the  description.